junction/proposals/urls.py

```diff
@@ -26,7 +26,7 @@
     url(r'^(?P<slug>[\w-]+)/delete/$', views.delete_proposal, name='proposal-delete'),
     url(r'^(?P<slug>[\w-]+)/update/$', views.update_proposal, name='proposal-update'),
     url(r'^(?P<slug>[\w-]+)/upload-content/$', views.proposal_upload_content, name='proposal-upload-content'),
-    # url(r'^(?P<slug>[\w-]+)/review/$', views.review_proposal, name='proposal-review'),
+    url(r'^(?P<slug>[\w-]+)/change-proposal-review-state/$', views.review_proposal, name='proposal-review'),
```

Comment:
`review_proposal` view should be accessible to only `super user or staff`. You need to add the check on the view, as of now `reviewers` can change it.